### PR TITLE
Add Nokia X & XL support 

### PR DESCRIPTION
**Pull request by yshalsager**

### CODE REVIEW

**Comment by chenxiaolong** [comment]
@chenxiaolong-bot test this please

File: data/devices/nokia.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+- name: Nokia XL
+  id: normandy
+  codenames:
+    - normandy
+  architecture: armeabi-v7a
+
+   system:
```
Comment (chenxiaolong) [inline data/devices/nokia.yml]: This entire section (`system`, `cache`, `data`, `boot`, `recovery`) is indented one space too far. The build failed with:

```
nokia.yml: Failed to convert file: yaml-cpp: error at line 7, column 10: illegal map value
```

You can check the syntax at: http://www.yamllint.com/

Comment (yshalsager) [inline data/devices/nokia.yml]: Done. Sorry for errors I'm new with yml

**Comment by chenxiaolong** [comment]
@chenxiaolong-bot test this please

**Pull request merged**